.php_cs.dist

```diff
@@ -6,6 +6,8 @@ $finder = PhpCsFixer\Finder::create()
     ->in('src')
     ->notPath('Kernel.php');
 
+$cacheDir = getenv('TRAVIS') ? getenv('HOME') . '/.php-cs-fixer' : __DIR__;
```

Comment:
To be fair, I'd rather not have a reference to Travis in here. Also, isn't that the default value if we leave it empty?